tools/props/VersionsNet8OrLater.props

```diff
@@ -4,5 +4,7 @@
   <PropertyGroup>
     <SystemConfigurationConfigurationManagerVersion>8.0.0</SystemConfigurationConfigurationManagerVersion>
     <SystemRuntimeCachingVersion>8.0.0</SystemRuntimeCachingVersion>
+    <SystemTextEncodingsWebVersion>8.0.0</SystemTextEncodingsWebVersion>
```

Comment:
System.text.encodings.web is part of the shared framework since .NET 6, so no explicit refrence is needed